seattle_lidar/Seattle_Lidar.ipynb

```diff
@@ -0,0 +1,461 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%time display(tiles * plot(dataset))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%time display(tiles * plot(df_merc))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Visualize Raster of Point Data\n",
+    "\n",
+    "If we compute a raster of the point data then we don't need to store as much data in memory, which should allow faster interactions, and allow use with lots of other tools. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%time raster = plot(dataset, dynamic=False, width=1000, height=1000).data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The result is an `xarray.Dataset` with x and y coordinates and a 2D array of Z values."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "raster.metersnorth[1] - raster.metersnorth[0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With these data we can use the geo tools in datashader to compute and visualize the elevation using hillshading for instance. See [Datashader User Guide #8](https://github.com/pyviz/datashader/blob/master/examples/user_guide/8_Geography.ipynb) for more datashader geo tooling. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from datashader.geo import hillshade"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "illuminated = hillshade(raster.Z)\n",
+    "hv_shaded = hv.Image((raster.meterswest, raster.metersnorth, illuminated))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tiles * rasterize(hv_shaded.options(opts.Image(cmap='blues', height=600, width=600)))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Ideas for future work\n",
+    "\n",
+    "It'd be nice to have a `rasterio` writer from `xarray` so that we could easily write chunked geotiffs from `xarray` objects.\n",
+    "\n",
+    "Something like:\n",
+    "\n",
+    "```python\n",
+    "raster.to_rasterio(path=None, mode='w', compute=True)\n",
+    "```"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.0"
```

Comment:
Did you mean for this metadata to be included?

Reply:
Fixed in d1e28bf